gtsfm/densify/mvsnets/methods/PatchmatchNet/eval_gtsfm.py

```diff
@@ -0,0 +1,313 @@
+"""Customized MVSNet evaluation functions for gtsfm
+    reference: https://github.com/FangjinhuaWang/PatchmatchNet
```

Comment:
Ren, if you run `black  gtsfm/densify/mvsnets/methods/PatchmatchNet/eval_gtsfm.py -l 120` this file will already be a lot cleaner